### PR TITLE
Readahead ubi8.4

### PR DESCRIPTION
**Pull request by vikasgupta8**
Build script for package readahead.

### CODE REVIEW

File: r/readahead/readahead_ubi8.4.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : github.com/klauspost/readahead
```
Comment (amitsadaphule) [inline r/readahead/readahead_ubi8.4.sh:3]: This field is for package name, in this case, readahead. Please update.

File: r/readahead/readahead_ubi8.4.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+# Version       : v1.3.1
+# Source repo   : https://github.com/klauspost/readahead
+# Tested on     : RHEL 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : BulkPackageSearch Automation <sethp@us.ibm.com> Vikas Gupta <vikas.gupta8@ibm.com>
```
Comment (amitsadaphule) [inline r/readahead/readahead_ubi8.4.sh:8]: Please correct the Maintainer field. Should be `# Maintainer    : Vikas Gupta <vikas.gupta8@ibm.com>`

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: r/readahead/readahead_ubi8.4.sh (hunk 1 above)
Comment (vikasgupta8) [reply r/readahead/readahead_ubi8.4.sh]: done

File: r/readahead/readahead_ubi8.4.sh (hunk 2 above)
Comment (vikasgupta8) [reply r/readahead/readahead_ubi8.4.sh]: done.

**Pull request merged**